File: CTDK_Python/115/_115.py
n = int(input("Nhap n: "))
if (n == 0):
    ahh = -1
elif (n == 1):
    ahh = 3
else:
    att = -1
    at = 3
    i = 2

    while (i <= n):
        ahh = 5 * at + 6 * att
        i += 1
        att = at
        at = ahh

print("Ket qua:", ahh)

File: CTDK/115/115.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	int ahh = 0;
	int att = -1;
	int at = 3;
	int i = 2;
	while (i <= n)
	{
		ahh = 5 * at + 6 * att;
		i++;
		att = at;
		at = ahh;
	}
	cout << "So hang thu " << n << " : " << ahh << endl;
	return 0;
}